spec.html

```diff
@@ -284,6 +284,12 @@ <h3>Non-terminals</h3>
                     <td class="fix ex">"\x06"</td>
                     <td><a href="https://github.com/mongodb/specifications/blob/master/source/client-side-encryption/subtype6.rst">Encrypted BSON value</a></td>
                   </tr>
+                  <tr>
+                    <td class="fix nt"></td>
+                    <td class="fix op">|</td>
+                    <td class="fix ex">"\x07"</td>
+                    <td>Compressed Column</td>
```

Comment:
I wonder if this should say "Compressed BSON column" (compare with "Encrypted BSON value"). After all, we store a column of arbitrary BSON, not just some numerical type or such.

Reply:
I am fine with that, made the change.